src/metaschema/oscal_metadata_metaschema.xml

```diff
@@ -751,6 +754,62 @@
             </index-has-key>
         </constraint>
     </define-assembly>
+ 
+    <define-assembly name="action">
+        <formal-name>Action</formal-name>
+        <description>An action applied by a role within a given party to the content.</description>
+        <define-flag name="uuid" as-type="uuid" required="yes">
+            <formal-name>Action Universally Unique Identifier</formal-name>
+            <description>A unique identifier that can be used to reference this defined action elsewhere in an OSCAL document. A UUID should be consistently used for a given location across revisions of the document.</description>
+        </define-flag>
+        <define-flag name="role-id" as-type="token" required="yes">
+            <formal-name>Action Responsible Role Identifier</formal-name>
+            <description>Identifier of the approving role in the responsible party.</description>
+        </define-flag>
+        <define-flag name="party-uuid" as-type="token" required="yes">
+            <formal-name>Action Party Universally Unique Identifier</formal-name>
+            <description>The universally unique identifier of the party responsible for the action.</description>
+        </define-flag>
```

Comment:
For consistency, it might be better to use `responsible-party` here instead of these. Thoughts?

Reply:
I can agree to that.